.github/workflows/nix-dev-cache.yaml

```diff
@@ -38,3 +43,5 @@ jobs:
         authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
     - name: build all packages and development shells
       run: nix -L build --accept-flake-config --no-link --keep-going '.#all'
+    - name: print disk free status
+      run: df -h
```

Comment:
I’m guessing this is intentional and not just leftover debugging.

Reply:
Yeah I added this intentionally for future debugging

Reply:
This seems super weird to me. /mnt is only supposed to be used transiently by a system administrator, not something that should be scripted against. There is no reason (if you care about the LFS) that there should be anything mounted to /mnt to make it a separate filesystem than /. Is /mnt just some other big disk on GHA?

If this works, that's great, I'm just curious, as this seems kinda wrong to me...